src/coreneuron/permute/data_layout.hpp

```diff
@@ -10,7 +10,3 @@
 
 #define SOA_LAYOUT 0
 #define AOS_LAYOUT 1
-namespace coreneuron {
-struct Memb_list;
-int get_data_index(int node_index, int variable_index, int mtype, Memb_list* ml);
```

Comment:
Do we not need this function in the header anymore? I see it's no longer part of any header actually; any reason for this?

Reply:
I thought the 2 defines were still necessary. I can try to remove it

Reply:
yes, imported end used in `phase2.cpp` and `capac.cpp` for example